scope/src/doctor/runner.rs

```diff
@@ -230,6 +246,17 @@ where
     }
 }
 
+fn prompt_user(prompt_text: &str) -> bool {
+    tracing_indicatif::suspend_tracing_indicatif(|| {
+        inquire::Confirm::new(prompt_text)
+            .with_default(false)
+            //FIXME: do we want to allow a help message here?
+            // .with_help_message("additional text explaining why this needs approval")
```

Comment:
Need to decide on this now because changing it would be a breaking change.

Reply:
I decided to just go ahead and provide the mechanism to display an additional help message.